src/OmniSharp.Shared/FileSystem/FileSystemHelper.cs

```diff
@@ -42,6 +44,27 @@ public IEnumerable<string> GetFiles(string includePattern, string targetDirector
             return matcher.GetResultsInFullPath(targetDirectory);
         }
 
+        public Matcher BuildMatcher()
+        {
+            Matcher matcher = new();
+            matcher.AddIncludePatterns(new string[] { "./**/*", "./*" });
```

Comment:
I've changed exclude to include.
Now I can get diagnostics from documents outside of current directory.
But now there is opposite problem: documents outside of current directory can not be ignored via options :D

Reply:
For some reason Matcher doesn't work well will relation pathes outside current directory.
But I don't think this is huge problem since ignoring documents outside directory of current solution is not frequent case.